themes/default/build/sass/utilities/_mixins.scss

```diff
@@ -2,73 +2,31 @@
  * A file for Custom mixins and placeholders
  */
 
-// Responsive Mixins
+// Responsive Mixin
 
-// NOTE: the first two media queries are to limit styles to mobile sizes
-// xs doesn't need a media query to apply styles - they will usually
-// be the default
-@mixin respond($media) {
+// NOTE: xs doesn't need a media query - mobile styles are the default
 
-	@if $media == xs__max {
-		@media screen and (max-width: $respond-xs - 0.01) {
-			@content;
-		}
-	}
-
-	@if $media == sm__max {
-		@media screen and (max-width: $respond-sm - 0.01) {
-			@content;
-		}
-	}
-
-	// everything up to regular tablet size
-	@if $media == md__max {
-		@media screen and (max-width: $respond-md - 0.01) {
-			@content;
-		}
-	}
-
-	@if $media == lg__max {
-		@media screen and (max-width: $respond-lg - 0.01) {
-			@content;
-		}
-	}
+// Usage example: @include respond(xl) {} 
+// Usage example: @include respond(md, max) {}
+// Usage example: @include respond(print) {}
 
+// _mixins.scss
 
-	@if $media == xl__max {
-		@media screen and (max-width: $respond-xl - 0.01) {
-			@content;
-		}
-	}
+@mixin respond($breakpoint, $maxmin:'min') {
```

Comment:
I prefer this as explicit and declarative. Loops are useful, but they make it much harder to read, and a little harder to use. It should be obvious that people can add their own media queries here if they need to.